packages/flutter_test/lib/src/key_labels.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// DO NOT EDIT -- DO NOT EDIT -- DO NOT EDIT
+// This file is generated by dev/tools/gen_keycodes/bin/gen_keycodes.dart and
+// should not be edited directly.
+//
+// Edit the template dev/tools/gen_keycodes/data/key_labels.tmpl instead.
+// See dev/tools/gen_keycodes/README.md for more information.
+
+/// The keyLabel corresponding to a logical key.
+///
+/// Used to generate simulated key events out of a logical key.
+const Map<int, String> logicalKeyLabels = <int, String>{
+  0x00000000061: r'a',
```

Comment:
If all of these are just Unicode keys, why do we need the lookup? I mean, some of them are numerical keyboard keys, so those make sense, but the ASCII (or really, most printable keys) can be generated.

Reply:
You're right. This is legacy code that I forgot to remove. I've removed them in favor for `key.keyLabel`.